ci/scripts/manage-versions.mjs

```
@@ -0,0 +1,111 @@
+  runCommand(
+    `cd transports && go get github.com/maximhq/bifrost/core@${version}`
+  );
+  runCommand("cd transports && go mod tidy");
+}
+
+// Main logic
+let result = {};
+
+switch (triggerType) {
+  case "core":
+    const coreVersion = inputVersion;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)